resources/assets/js/features/user/reducer.js

```diff
@@ -48,6 +53,45 @@ export default function (state = initialState, action) {
                     errors: {}
                 }
             };
+        case actions.LOGIN_SUCCESS:
+            browserHistory.push('/dashboard')
+            return {
+                ...state,
+                login: {
+                    success: !!sessionStorage.jwt
```

Comment:
all work with session should be in AuthManager/AuthService, it should know how to verify if session active or not